src/k8s-extension/azext_k8s_extension/partner_extensions/Dapr.py

```diff
@@ -144,3 +145,39 @@ def Create(self, cmd, client, resource_group_name: str, cluster_name: str, name:
             location=""
         )
         return extension_instance, release_name, create_identity
+
+    def Update(self, cmd, resource_group_name: str, cluster_name: str, auto_upgrade_minor_version: bool,
+               release_train: str, version: str, configuration_settings: dict,
+               configuration_protected_settings: dict, original_extension: Extension, yes: bool = False) \
+            -> PatchExtension:
+        """ExtensionType 'Microsoft.Dapr' specific validations & defaults for Update.
+           Must create and return a valid 'PatchExtension' object.
+        """
+        input_configuration_settings = deepcopy(configuration_settings)
+
+        # configuration_settings can be None, so we need to set it to an empty dict.
+        if configuration_settings is None:
+            configuration_settings = {}
+
+        # If we are downgrading the extension, then we need to disable the apply-CRDs hook.
+        # This is because CRD updates while downgrading can cause issues.
```

Comment:
nit: may be add some info here like - it is not recommended to remove or update crd while downgrading. It is better to keep append only behavior ?

Reply:
Do you want to add a prompt here for the user, or only comment?

Reply:
only comment.

Reply:
Ok, added another line to clarify, let me know if you want any other updates!